src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3362,24 +3431,71 @@ private TypeSymbolWithAnnotations VisitOptionalImplicitConversion(BoundExpressio
 
         public override BoundNode VisitTupleLiteral(BoundTupleLiteral node)
         {
-            VisitTupleExpression(node);
+            VisitTupleExpression(node, node.ArgumentNamesOpt);
             return null;
         }
 
         public override BoundNode VisitConvertedTupleLiteral(BoundConvertedTupleLiteral node)
         {
-            VisitTupleExpression(node);
+            VisitTupleExpression(node, default);
             return null;
         }
 
-        private void VisitTupleExpression(BoundTupleExpression node)
+        private void VisitTupleExpression(BoundTupleExpression node, ImmutableArray<string> argumentNamesOpt)
```

Comment:
>argumentNamesOpt [](start = 92, length = 16)

Parameter seems superfluous (why would tuple element names matter in nullable analysis?)

Reply:
Good catch, thanks. This was left over from an earlier iteration.

---
In reply to: [249161045](https://github.com/dotnet/roslyn/pull/32612#discussion_r249161045) [](ancestors = 249161045)